snippets/assert-attribute-must-be-set.js

```diff
@@ -0,0 +1,48 @@
+/* Usage of this snippet:
+ *   In the case when you want to confirm an element having a specified attribute.
+ *
+ *   Change the values written below.
+ *    selector: A string of a selector to pinpoint the element.
+ *    attrName: A string of the attribute which you want to assert its value.
+ *    errMessage:
+ *      A string of the error message which you want to show when it fails.
+ *      It will be a default string if you don't change from <TODO: REPLACE>.
+ *
+ * このスニペットの使い方:
+ *   要素が特定の属性を持っていることを確認をしたいときにご利用ください。
+ *
+ *   以下の値を変更してください
+ *    selector: 対象要素を特定するセレクタの文字列
+ *    attrName: 値を検証したい属性の文字列
+ *    errMessage:
+ *      検証が失敗したときに表示したいエラーメッセージの文字列
+ *      `<TODO: REPLACE>` のまま変更がない場合はデフォルトの文言で表示します。
+ */
+
+var selector = "<TODO: REPLACE>";
+var attrName = "<TODO: REPLACE>";
+var errMessage = "<TODO: REPLACE>";
+
+
+/* --------- You don't need to touch below ---------------
+ * --------- ここから下は変える必要はありません ---------- */
+
+/* Locate the element
```

Comment:
コメントの方法について完全に好みの問題なんですが、ちょっと違和感があります
```suggestion
/**
 *  Locate the element
```

Reply:
ここは本当は `//` で書きたいところだけど、 `//` で書くとIEが 落ちるからしょうがなく複数行コメントの書き方にしてるだけでコメントのために無駄に行数増やしたいわけではないんですよねぇ

Reply:
複数行で `/* ... */` という書き方をするとエディタによっては警告が出てしまうため、メジャーな書き方に合わせて 一括で `/** ... */` の形式に修正しました。

単行で `//` が使えない場合、行ごとに `/* ... */`  を使うという手もあります。

```
// これはOK（だけど現在IEでエラーが出ます）
/* これもOK */

/**
 * これも
 * OK
 */

/* これは
 * エディタによっては
 * 警告が出ます */